Cubical/Algebra/Group/Properties.agda

```diff
@@ -1,8 +1,3 @@
-{-
```

Comment:
Why remove comment?

Reply:
It is a pleonasm. I don't see the point to add something already  in the name of the file.
But it can be keept too, it doesn't really matter to me